x/gamm/client/cli/cli_test.go

```diff
@@ -159,6 +171,119 @@ func (s *IntegrationTestSuite) TestNewCreatePoolCmd() {
 			},
 			false, &sdk.TxResponse{}, 5,
 		},
+		{
+			"future governor address",
+			[]string{
+				fmt.Sprintf("--%s=%s", cli.FlagWeights, "1node0token,3stake"),
+				fmt.Sprintf("--%s=%s", cli.FlagInitialDeposit, "100node0token,100stake"),
+				fmt.Sprintf("--%s=%s", cli.FlagSwapFee, "0.001"),
+				fmt.Sprintf("--%s=%s", cli.FlagExitFee, "0.001"),
+				fmt.Sprintf("--%s=%s", cli.FlagFutureGovernor, "cosmos1fqlr98d45v5ysqgp6h56kpujcj4cvsjn6mkrwy"),
+				fmt.Sprintf("--%s=%s", flags.FlagFrom, newAddr),
+				// common args
+				fmt.Sprintf("--%s=true", flags.FlagSkipConfirmation),
```

Comment:
TODO for the future, we should refactor out these common args into another variable, and have them be used in the test cases automatically added.